app/views/content/non-uk-teachers/promos/_adviser-promo-training-in-england.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Teacher training advisers", classes: %w[tta-background]) do %>
+  <% end %>
+
+  <% promo.right_section(
+    heading: "Get free one-to-one support"
+  ) do %>
+      <p>An adviser can answer all your questions about training to teach in England, what it's like in an English classroom, and support your application.</p>
```

Comment:
I don't think this quite makes sense - the last part doesn't fit with 'answer all your questions'? Does it need breaking into two sentences? Or just rewording?

I also think we ought to slightly careful about saying 'support your application' - it's more that they will support you **with** your application? The way it is written currently sounds like they will advocate for you with providers?